x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -173,15 +173,20 @@ constant
     | decimalValue                                                                      #decimalLiteral
     | integerValue                                                                      #integerLiteral
     | booleanValue                                                                      #booleanLiteral
-    | PARAM                                                                             #inputParam
+    | params                                                                            #inputParams
```

Comment:
I got this error after removing #inputParams method for params, seems like it has to be there because all of its siblings have methods, will leave it there even we don't do anything about it.

```
> Task :x-pack:plugin:esql:regenParser FAILED
error(122): EsqlBaseParser.g4:162:0: rule constant: must label all alternatives or none
```